Libraries/Lists/FlatList.js

```diff
@@ -606,24 +606,71 @@ class FlatList<ItemT> extends React.PureComponent<Props<ItemT>, void> {
           return (
             <View style={StyleSheet.compose(styles.row, columnWrapperStyle)}>
               {item.map((it, kk) => {
-                const element = renderer({
-                  item: it,
-                  index: index * numColumns + kk,
-                  separators: info.separators,
-                });
+                const accessibilityCollectionItemInfo = {
+                  rowIndex: index,
+                  rowSpan: 1,
+                  columnIndex: (index * numColumns + kk) % numColumns,
+                  columnSpan: 1,
+                  heading: false,
+                  itemIndex: index * numColumns + kk,
+                };
+
+                const element = (
+                  <View
+                    importantForAccessibility="yes"
+                    style={styles.cellStyle}
+                    accessibilityCollectionItemInfo={
```

Comment:
The only issue I can think of with this approach is that it needs an additional wrapper View so that we can get the accessibility collection info on the native side. 
The solution would be if we can create accessibility-only Views. I think @amarlette had created an issue for the same, but I am able to find it.